lib/matplotlib/tests/test_determinism.py

```diff
@@ -67,12 +145,13 @@ def _save_figure(objects='mhi', fmt="pdf", usetex=False):
         ("m", "pdf", False),
         ("h", "pdf", False),
         ("i", "pdf", False),
-        ("mhi", "pdf", False),
-        ("mhi", "ps", False),
+        ("p", "svg", False),  # (clipping) paths are only relevant for SVG output
```

Comment:
A late self-nitpick here: this comment seems poorly phrased and could potentially be misleading to future readers.

If I understand correctly, _Matplotlib_ clipping is a feature that is output-format-agnostic (that is, it should work for all output formats).

The fix in this pull request only affects _SVG_ elements named `clipPath` -- but they're a different concept.

I do think it could make sense to perform an isolated SVG-format-only path test (`p`), alongside the complete-functionality test (`mhip`) -- but I think the comment attempting to explain should either be improved, or omitted entirely.

At the moment I'm leaning towards removing it entirely, and perhaps relocating the line so that future readers don't consider it a possible typo/accidental difference from the preceding `pdf` test parameters.

(maybe an overly verbose explanation for a small detail, but I want to try to explain my thinking)

Reply:
(done, and no further changes planned on this branch)